modules/manage/pages/audit-logging.adoc

```diff
@@ -0,0 +1,46 @@
+= Audit Logging
+:description: Learn how to use Redpanda's audit logging capabilities.
+
+Many scenarios for streaming data include the need for fine-grained auditing of user activity related to the system. This is especially true for regulated industries such as finance, healthcare, and the public sector. Complying with PCI DSS v4 standards, for example, requires verbose and detailed activity auditing, alerting, and analysis capabilities.
+
+Redpanda's auditing capabilities support auditing both administrative and operational interactions with topics and with users. We comply with the open cybersecurity schema framework, providing a predictable and extensible solution which works seamlessly with industry standard tools.
+
+NOTE: All audit logging is configured at the Cluster level. Audit logging is disabled by default.
+
+== Audit Log Flow
+
+The Redpanda audit log mechanism functions similar to the Kafka flow you may be familiar with. When users interact with other users or with topics, either with administrative functions or user functions, Redpanda writes an event to a specialized audit topic. The audit topic is immutable to everything except Redpanda iself -- users are prevented from writing to the topic directly and the Kafka API cannot create or delete it.
```

Comment:
I would drop the `to everything`. Instead, something like "The audit topic is immutable, such that only Redpanda itself may write to this topic--users are ..."